### PR TITLE
TASK: Fix code example in CustomTypoScriptObjects docs

### PR DESCRIPTION
**Pull request by aertmann**

### CODE REVIEW

File: Neos.Neos/Documentation/ExtendingNeos/CustomTypoScriptObjects.rst [hunk 1]
```diff
@@ -25,15 +25,20 @@ extended. The only method that needs to be implemented is ``evaluate()``. To acc
...
-	use Neos\Fusion\FusionObjects\AbstractFusionObject
+	use Neos\Fusion\FusionObjects\AbstractFusionObject;
 
 	class GravatarImplementation extends AbstractFusionObject {
+
+		/**
+		 * @return string
+		 */
 		public function evaluate() {
 			$emailAddress = $this->tsValue('emailAddress');
```
Comment (kitsunet) [inline Neos.Neos/Documentation/ExtendingNeos/CustomTypoScriptObjects.rst]: While at it, tsValue => fusionValue as the other is deprecated anyway.

Comment (aertmann) [inline Neos.Neos/Documentation/ExtendingNeos/CustomTypoScriptObjects.rst]: fixed

**Comment by kitsunet** [comment]
Much appreciated :)

**Comment by aertmann** [comment]
thanks for reviewing

**Pull request merged**